core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -160,6 +166,7 @@ public boolean equals(Object object) {
         return Objects.equals(type, clusterConfig.type)
                 && Objects.equals(brokers, clusterConfig.brokers)
                 && Objects.equals(controllers, clusterConfig.controllers)
+                && Objects.equals(disksPerBroker, clusterConfig.disksPerBroker)
```

Comment:
not sure whether we need to keep `hashCode` and `equals`. We don't use it actually and it is error-prone when adding new fields. For example, this PR does not update `hashCode` for the new field. @brandboat WDTY?

Reply:
feel free to remove them as currently they are unused 😃

Reply:
Remove `hashCode` and `equals` functions. Thanks.